### PR TITLE
Pin Helm version to avoid 400 Bad Request error

### PR DESCRIPTION
**Pull request by wjayesh**
## Describe changes
I fixed the publish helm chart action by pinning the helm version in the runner to one that works for me locally.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/stacks-and-components/component-guide) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.
- [ ] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: .github/workflows/publish_helm_chart.yml [hunk 1]
```diff
@@ -42,6 +42,12 @@ jobs:
           mask-password: 'true'
           registry-type: public
 
+      - name: install helm with version greater than v3.13.1
+        uses: azure/setup-helm@v3
+        with:
+          version: "v3.9.2" # default is latest (stable)
```
Comment (strickvl) [inline .github/workflows/publish_helm_chart.yml:48]: how is 3.9.2 greater than 3.13.1?

Comment (wjayesh) [reply .github/workflows/publish_helm_chart.yml:48]: ah yes, sorry. wrong comment. i used 3.13.1 but that didn't work so I then decided to go with my local version

File: .github/workflows/publish_helm_chart.yml [hunk 2]
```diff
@@ -42,6 +42,12 @@ jobs:
           mask-password: 'true'
           registry-type: public
 
+      - name: install helm with version 3.9.2
```
Comment (strickvl) [inline .github/workflows/publish_helm_chart.yml:45]:
```suggestion
      #TODO: revisit the pinned helm version + find a higher version
      - name: install helm with version 3.9.2
```

**Review by strickvl — APPROVED** [review]
LGTM. Maybe worth adding a #TODO as I suggested, but otherwise looks fine.

**Review by safoinme — APPROVED** [review]

**Pull request merged**